packages/app/src/runner/SnapshotHighlightControls.vue

```diff
@@ -6,32 +6,26 @@
     >{{ t('runner.snapshot.toggleHighlights') }}</label>
     <Switch
       name="toggle-highlights"
-      :value="toggleHighlights"
+      :value="props.value"
       size="sm"
-      @update="toggleHighlights = !toggleHighlights"
+      @update="emits('update:value', !props.value)"
```

Comment:
It doesn't look like we ever read the boolean value we emit here, can we remove the value from the API?